js/src/cover.js

```diff
@@ -53,27 +53,23 @@ jQuery(document).ready(function() {
         }
     });
 	
-	/*
-	 * Skrollr and Headroom.
+    /**
+     * Headroom.
+     */
+    var header_headroom = new Headroom(jQuery('.header')[0]);
```

Comment:
Identifier 'header_headroom' is not in camel case.<br>'Headroom' is not defined.